porter/v1/cluster_control_plane.proto

```diff
@@ -742,3 +745,17 @@ message DockerConfigFileForRegistryResponse {
   option deprecated = true;
   bytes docker_config_file = 1;
 }
+
+message ClusterNetworkSettingsRequest {
+  EnumCloudProvider cloud_provider = 1;
```

Comment:
It makes more sense to have this in the response, as opposed to the request. Backend can always deduce the provider from the cluster id